src/lib/Components/Inbox/Conversations/Messages.tsx

```diff
@@ -104,6 +105,7 @@ export class Messages extends React.Component<Props, State> {
         inverted={!this.state.shouldStickFirstMessageToTop}
         data={this.state.shouldStickFirstMessageToTop ? messages.reverse() : messages}
         renderItem={this.renderMessage.bind(this)}
+        keyExtractor={(item, index) => index.toString()}
```

Comment:
Wat's this?

Reply:
Any React list needs to have keys to say which ones can be updated, this says how you can get a unique ID per item, index isn't exactly optimal TBH

Reply:
Oh you mean to avoid the `toString`? Will update 👍

Reply:
> keyExtractor tells the list to use the ids for the react keys instead of the default key property.

In that case, setting the index as the unique key would mean that if the data changes (like if you reloaded the data) React would use the index as the unique key to decide if something changed. Ideally you'd use the represented object's UUID. So I _think_ `item` here would be the message, so you'd want the `.id` on the message to be the unique key for that flattest cell.